packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -208,34 +219,38 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
     }
   }, [isOpen]);
 
-  // Initialize filtered options to the latest value options
-  const [filteredOptions, setFilteredOptions] = React.useState(options);
-  React.useEffect(() => {
-    setFilteredOptions(options);
-    dispatch(['focus']);
-  }, [options]);
-
   // Filter options based on input value
```

Comment:
This comment is out of date now because there is no filtering going on

Reply:
Removed it, now it is pretty straight forward what it does